README.md

```diff
@@ -17,44 +17,34 @@ Note: The PRM and Bimanual reproductions do not yet work on Deepnote and the UAV
 
 ## Running locally
 
-### Installing Dependencies
```

Comment:
Do we not want to support anything else except the docker workflow?  I guess that could be reasonable...

Reply:
I re-added the other workflows. Although for reproducibility, docker is a much better option.